src/main/java/io/github/acm19/aws/interceptor/http/AwsRequestSigningApacheInterceptor.java

```diff
@@ -80,13 +81,13 @@ public AwsRequestSigningApacheInterceptor(String service,
      */
     @Override
     public void process(HttpRequest request, HttpContext context)
-            throws HttpException, IOException {
```

Comment:
8 spaces of indentation is more conventional for wrapping a line: https://www.oracle.com/technetwork/java/codeconventions-150003.pdf

Same comments for all other wrapping adjustment to avoid keep repeating the same comment.